core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Sample.java

```diff
@@ -62,6 +64,8 @@
   public static final Long UNSAVED_ID = 0L;
   /** Field PREFIX */
   public static final String PREFIX = "SAM";
+  List<String> CATEGORIES = Arrays.asList(SampleIdentity.CATEGORY_NAME, SampleTissue.CATEGORY_NAME,
```

Comment:
Should be immutable (`Arrays.asList` returns a mutable `List`). SampleCategory should probably become an enum at this point. Should be on `SampleClass`, or at least `DetailedSample` rather than plain `Sample` if it stays within another class